### ISSUE TITLE
Question about 'roslaunch launch/just_robot.launch'

### ISSUE THREAD

**Issue by ZhengLeon** [IRVLUTD/SceneReplica#1]
Dear @kninad,
Thanks for your great work and for sharing very much.
I want to try the gazebo demo but can't find the `just_robot.launch` file.
Could you please give me some guide on how to set up it?

Best,
Leon

**Comment by kninad** [comment]
Hi Leon, thanks for pointing this out! The launch file is prettty simple, it just launches the Fetch robot in the simulation environment. We will shortly upload it to the main branch along with other helpful launch files in the `launch/` folder :)

**Comment by 07hokage** [comment]
Hi @ZhengLeon , please find the launch files here: [https://github.com/IRVLUTD/SceneReplica/tree/main/launch](https://github.com/IRVLUTD/SceneReplica/tree/main/launch). They have been updated in the repository.

**Comment by ZhengLeon** [comment]
Hi @07hokage , thanks for your update. However, I find another issue that the `fetch_gazebo` package can't be installed in ubuntu20.04. May I ask what the tested environment in this project? Thanks.

**Comment by 07hokage** [comment]
@ZhengLeon , the setup instructions can be found here [https://github.com/IRVLUTD/SceneReplica/tree/main#ros-setup](https://github.com/IRVLUTD/SceneReplica/tree/main#ros-setup) i.e., in the main Readme description. Once the setup is done, the SceneReplica repo can be cloned and the simulation can be run as mentioned in [here](https://github.com/IRVLUTD/SceneReplica/#gazebo-simulation-usage). Let us know how it goes.

**Comment by ZhengLeon** [comment]
Hi @07hokage , thanks a lot, it works well now.
